### ISSUE TITLE
Look into user research to support Process Page definition

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#817]
We want to do user/SME interviews to support #816. Ideally, find residents who are applying for mobile food permits or who have recently done so and learn about their approach, pain points, etc. 

Determine what that research should look like. Creating issues (plan/scheduling, interviews, documentation/synthesis) from there. 

Blocked by https://github.com/cityofaustin/odd-ops/issues/101 since it is not a good use of time to plan the interviews if we can't compensate people.

**Comment by amenity** [comment]
Will need to this at some point, but we're pivoting towards permitting content instead of mobile food for our first process pages.

**Comment by amenity** [comment]
Decided to create Information Page (#867) and Service Collection (#891) content types. No longer relevant.
